Sources/SmileID/Classes/DocumentVerification/Model/DocumentCaptureModel.swift

```diff
@@ -50,6 +50,11 @@ class DocumentCaptureViewModel: ObservableObject, JobSubmittable, ConfirmationDi
     private var files = [URL]()
     private var textDetected = false
     private var recieveBufferQueue = DispatchQueue(label: "com.smileid.receivebuffer")
+    private var documentCaptureStart = 0
+    private var waitTimeInSecs = 10
+    var autoCaptureTimer: Timer?
+    var enableManualCaptureTimer: Timer?
```

Comment:
nit: the naming of this sounds like it would be a boolean to me, was slightly confusing to read